assets/search/components/SearchResultsBar/SearchResultsAdvancedSearchRow.tsx

```diff
@@ -70,103 +83,36 @@ export function SearchResultsAdvancedSearchRow({
         }
     };
 
-    if (keywords.all.length) {
-        advancedSearchTags.push(
-            <Tag
-                key="tags-advanced--and"
-                text={gettext('and')}
-                operator={true}
-                shade="success"
-                readOnly={true}
-            />
-        );
-
-        keywords.all.forEach((term, index) => {
-            advancedSearchTags.push(
-                <Tag
-                    key={`tags-advanced--and-${index}`} // using index instead of term which might not be unique
-                    text={term}
-                    shade="success"
-                    readOnly={readonly}
-                    onClick={(event) => {
-                        event.preventDefault();
-                        removeKeywordEntry('all', index);
-                        refresh?.();
-                    }}
-                />
-            );
-        });
-    }
-
-    if (keywords.any.length) {
-        if (advancedSearchTags.length) {
-            advancedSearchTags.push(
-                <span
-                    key="tag-advanced--separator-any"
-                    className="tag-list__separator"
-                />
-            );
-        }
-
-        advancedSearchTags.push(
-            <Tag
-                key="tags-advanced--or"
-                text={gettext('or')}
-                operator={true}
-                shade="info"
-                readOnly={true}
-            />
-        );
-
-        keywords.any.forEach((term, index) => {
+    for (const field of Object.keys(keywords) as Array<keyof typeof keywords>) {
+        // due to the declaration outside key would be the last key
```

Comment:
since you fixed the problem with scope, the comment is no longer correct and should be removed